java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -2648,16 +2608,34 @@ private static List<Arguments> getStreamAddOptions() {
     }
 
     @SneakyThrows
-    @ParameterizedTest
+    @ParameterizedTest(name = "{0}")
     @MethodSource("getStreamAddOptions")
-    public void xadd_with_options_returns_success(Pair<StreamAddOptions, String[]> optionAndArgs) {
+    public void xadd_with_options_to_arguments(
+            String testName, StreamAddOptions options, String[] expectedArgs) {
+        assertArrayEquals(expectedArgs, options.toArgs());
+    }
+
+    @SneakyThrows
+    @Test
+    public void xadd_with_nomakestream_maxlen_options_returns_success() {
         // setup
         String key = "testKey";
         Map<String, String> fieldValues = new LinkedHashMap<>();
         fieldValues.put("testField1", "testValue1");
         fieldValues.put("testField2", "testValue2");
+        StreamAddOptions options =
```

Comment:
you can use empty options here to simplify the test

Reply:
🤷